src/checks.rs

```diff
@@ -28,22 +29,34 @@ fn checklist() -> Vec<Box<dyn Check>> {
     ]
 }
 
+fn ending_line_checker() -> Box<dyn Check> {
+    Box::new(ending_blank_line::EndingBlankLineChecker::default())
+}
+
+fn check_and_push_warning(checker: &mut dyn Check, line: &LineEntry, warnings: &mut Vec<Warning>) {
+    if let Some(warning) = checker.run(line) {
+        warnings.push(warning);
```

Comment:
Moving this code to a separate function is not necessary.

Reply:
Now we have this code in 2 places, and it seems that there will be more such places if we add new checks for empty lines or comments.

What do you think?

Reply:
I think there will be no more such places, so I would not like to move this code to the separate function.

Reply:
Done